stanford/roles/cluster/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Lookup/create subnet
+  debug:
+    msg: 'TODO: this'
+- name: Lookup/create security groups
+  debug:
+    msg: 'TODO: this'
+- name: Lookup/create load balancer
+  ec2_elb_lb:
+    state: present
+    scheme: internal
```

Comment:
This shouldn't be hard-coded;
it should be pulled from the `CLUSTER_ELBS` dictionary.